app/views/pages/components/author-profile/page-content.ejs

```diff
@@ -14,6 +14,9 @@
     <%= user.bio %>
   </div>
   <h2 class='author-profile__articles-header'>Author's articles</h2>
+  <% if(user._id.equals(currentUser?._id)) { %>
+    <a class='footer__unsubscribe-anchor' href='/articles/drafts'>My Drafts</a>
```

Comment:
Can u use different CSS class for this element? The name is misleading.

Reply:
Sure thing